docs/analysis.rst

```diff
@@ -167,25 +167,49 @@ Each of the width analysis functions are applied to this spectrum below:
 Template Comparison
 -------------------
 
-With one observed spectrum and n template spectra, the process to do template matching is:
-
-    1. Move the templates as close as possible to the observed spectrum by doing redshifting, matching the resolution,
-       and matching the wavelength spacing
-    2. Then loop through all of the n template spectra and run chi square on them using the observed spectrum as the
-       expected frequency and the template spectrum as the observed
-    3. Once you have a corresponding chi square for each template spectrum, return the lowest chi square and its
-       corresponding template spectrum (normalized) and the index of the template spectrum if the original template
-       parameter is iterable
-
-An example of how to do template matching in is:
+The ~`specutils.analysis.template_comparison.template_match` function takes one
```

Comment:
```suggestion
The ~`specutils.analysis.template_comparison.template_match` function takes an
```
(because in the future it may not really be just one)